### PR TITLE
Game and board

### PR DESCRIPTION
**Pull request by JackCSweeney**
Add Game class with following methods:
- Initialize
- Board
- Welcome Message

Add Board class with following methods:
- Initialize
- Add piece (still in progress)

### CODE REVIEW

File: lib/board.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    def initialize
+        @board = Array.new(6){Array.new(7, '.')}
+    end
+
+    def display
+        puts 'A B C D E F G'
+        @board.each {|row| puts row.join(' ')}
+    end
+
+    def add_piece(column, piece)
```
Comment (IgorrMagalhaess) [inline lib/board.rb:14]: This looks good. I thought about using rows = board.length -1 and count down to iterate over the rows.

File: lib/game.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+require './lib/board'
+class Game
+
+    attr_reader :board
+
+    def initialize
+        @board = Board.new
+    end
+
+    def welcome_message
```
Comment (IgorrMagalhaess) [inline lib/game.rb:10]: What if the welcome message is displayed with the start of the game

File: spec/board_spec.rb [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+            board = Board.new
+
+            board.add_piece('A', 'X')
+            board.add_piece('A', 'X')
+            board.add_piece('A', 'X')
+            board.add_piece('A', 'X')
+            board.add_piece('A', 'X')
+            board.add_piece('A', 'X')
+
+            expect(board.add_piece('A', 'X')).to eq(nil)
```
Comment (IgorrMagalhaess) [inline spec/board_spec.rb:45]: Great test, very robust

**Review by IgorrMagalhaess — APPROVED** [review]

File: lib/board.rb (hunk 1 above)
Comment (JackCSweeney) [reply lib/board.rb:14]: Interesting, let's try it out to see if it works more effectively for this method!

**Pull request merged**